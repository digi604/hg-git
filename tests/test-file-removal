#!/bin/sh

"$TESTDIR/hghave" git || exit 80

hg init
git init -q

echo alpha > alpha
git add alpha
git commit -qm 'add alpha'

echo beta > beta
git add beta
git commit -qm 'add beta'

mkdir foo
echo blah > foo/bar
git add foo
git commit -qm 'add foo'

git rm alpha
git commit -qm 'remove alpha'

git rm foo/bar
git commit -qm 'remove foo/bar'

echo % final manifest in git is just beta
git ls-files

hg gimport
hg log

echo
echo % make sure alpha is not in this manifest
hg manifest -r 3

echo
echo % make sure that only beta is in the manifest
hg manifest -r tip
