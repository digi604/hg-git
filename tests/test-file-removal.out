rm 'alpha'
rm 'foo/bar'
% final manifest in git is just beta
beta
importing Git objects into Hg
at: 0/5
creating bookmarks failed, do you have bookmarks enabled?
changeset:   4:26a36502e0eb
tag:         tip
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     remove foo/bar

changeset:   3:2649f8f87bc1
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     remove alpha

changeset:   2:16db36e9a614
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add foo

changeset:   1:c25ea3d53ce1
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add beta

changeset:   0:c50ac8b9444c
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add alpha


% make sure alpha is not in this manifest
beta
foo/bar

% make sure that only beta is in the manifest
beta
